mindsdb/integrations/handlers/rag_handler/rag_handler.py

```diff
@@ -133,6 +133,6 @@ def predict(self, df: pd.DataFrame = None, args: dict = None):
 
         # get question from sql query
         # e.g. where question = 'What is the capital of France?'
-        response = question_answerer(df["question"].tolist()[0])
+        response = question_answerer(df[args["input_column"]].tolist()[0])
```

Comment:
It should be `args.input_column`

Reply:
I had it at first. Changing it back.

Reply:
Ok. Now it works fine. Thanks.

![image](https://github.com/mindsdb/mindsdb/assets/109554435/d96ea8b0-ae1b-4bde-bf00-80be488e75b5)